modules/fb_instant_articles_display/fb_instant_articles_display.module

```diff
@@ -13,15 +13,15 @@
 */
 function fb_instant_articles_display_init() {
  // Configure Instant Articles Transformer logger to append to Drupal log if enabled
- $enable_transformer_logging = variable_get('fb_instant_articles_display_enable_transformer_logging', FALSE);
+ $enable_transformer_logging = variable_get('fb_instant_articles_display_enable_transformer_logging');
```

Comment:
@simonengelhardt Oh I removed that since the default 2nd param for `variable_get()` is `NULL`, so it still works the same.